tests/test_tests.py

```diff
@@ -0,0 +1,125 @@
+# placeholder file to test the testing capability
+
+import pytest
+
+from pyfecons import compute_cas22_reactor_equipment_total_cost
+from pyfecons.enums import (
+    ReactorType,
+    ConfinementType,
+    BlanketFirstWall,
+    BlanketType,
+    BlanketPrimaryCoolant,
+    BlanketSecondaryCoolant,
+    BlanketNeutronMultiplier,
+    BlanketStructure,
+)
+from pyfecons.inputs import Blanket, RadialBuild
+from pyfecons.units import Ratio, Meters
+
+
+def test_mfe_tokamak():
```

Comment:
Since we removed the TestCAS220101TotalCost class, I think it would be nice for the naming to reflect that this is just testing the cost calculation for CAS 22.1.1. Maybe `test_mfe_tokamak_cas220101_total_cost`?

Reply:
Done